packages/mux-player/src/index.ts

```diff
@@ -1,6 +1,6 @@
 import { globalThis, document } from './polyfills';
 import { MediaController } from 'media-chrome';
-import 'media-chrome/dist/experimental/media-captions-selectmenu.js';
+import 'media-chrome/dist/experimental/index.js';
```

Comment:
nitpick(blocking): We should directly import each experimental feature we use so we aren't unnecessarily adding to our bundles and are being explicit about the dependencies we are using.

Reply:
I had it first like that but then I thought what about custom themes.
it's nice to have all the experimental elements to drop in without needing imports.
the 1 left is playback-rate-selectmenu and dependencies, comparative to the Mux player bundle it's a drop in a bucket. thoughts?

Reply:
hmmm... I'm sold as solution for now. Maybe next week in media chrome office hours we can try to come up with a solution to make sure themes can include any of their dependencies? This has come up in the past and would hopefully avoid your concern here.